webapp/app.py

```diff
@@ -29,5 +29,13 @@
     provide_automatic_options=False,
 )
 
+app.add_url_rule(
+    "/notices/<notice_id>",
```

Comment:
Again, let's give this a `.json` endpoint, although fine for this to be in a follow-up PR.